dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSP23JSP22ServerTest.java

```diff
@@ -62,15 +67,17 @@ public static void testCleanup() throws Exception {
         /**
          * Stopping the server before changing the server features
          * makes CWWKZ0014W to not happen.
+         *
+         * Calling the full version of the stopServer method in order to skip repeat feature set checking
          */
         if (server != null && server.isStarted()) {
-            server.stopServer();
+            server.stopServer(LibertyServer.IGNORE_STOPPED, LibertyServer.POST_ARCHIVES, LibertyServer.FORCE_STOP, LibertyServer.SKIP_ARCHIVES, true,
```

Comment:
```suggestion
            server.stopServer(LibertyServer.IGNORE_STOPPED, LibertyServer.POST_ARCHIVES, LibertyServer.FORCE_STOP, LibertyServer.SKIP_ARCHIVES, LibertyServer.SKIP_FEATURE_CHECK,
```
After changing it to true as suggested in LibertyServer it will clearly state what the true means now.

Reply:
Done